src/utils/user/getters.ts

```diff
@@ -75,7 +75,13 @@ export const getUserByEmail = async (email: string) => {
   return users[0];
 };
 
-export const getUserByTicket = async (ticket: string) => {
+export const getUserByTicket = async (ticket: string | undefined) => {
```

Comment:
this change wouldn't be necessary anymore with the previous comment

Reply:
Done!